src/Microsoft.Health.Fhir.Core/Features/Definition/SearchParameterDefinitionManager.cs

```diff
@@ -261,10 +262,18 @@ private async Task LoadSearchParamsFromDataStore(CancellationToken cancellationT
                         catch (SearchParameterNotSupportedException ex)
                         {
                             _logger.LogWarning(ex, "Error loading search parameter {Url} from data store.", searchParam.GetStringScalar("url"));
+                            foreach (OperationOutcomeIssue issue in ex.Issues)
```

Comment:
I like the idea of adding more logs.
What is we create a StringBuilder, append all the errors together and log a single Warning? 
Probably all that information together in a single log entry will make more sense.

Reply:
Thank you for looking at this.